selenium-system-test/src/test/scala/hts/utils/Helpers.scala

```diff
@@ -17,9 +17,22 @@
 package hts.utils
 
 import hts.pages.Page
+import org.openqa.selenium.support.ui.{ExpectedConditions, WebDriverWait}
 import org.openqa.selenium.WebDriver
 
 object Helpers extends Page {
 
-  def isTextOnPage(text: String)(implicit driver: WebDriver): Boolean = text.r.findAllIn(getPageContent).nonEmpty
+  def isTextOnPage(text: String)(implicit driver: WebDriver): Boolean = {
```

Comment:
it'd be more informative to call the parameter `regex` rather than `text`

Reply:
also, when you're checking URLs you do a wait to check if the url is eventually the one you expect. When you're checking text on a page do you need to do something similar or do you want to check the condition once only?

Reply:
Once only for now.